src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private int leg;
+    private int base;
+
+    public IsoscelesTrapezoid(String color, String name) {

```

Comment:
The constructor seems to be missing parameters for 'base' and 'leg'. According to the checklist, you can pass random values to the constructor instead of generating them inside figure classes.